### ISSUE TITLE
Add +/- icons to the diff to improve accessibilty

### ISSUE THREAD

**Issue by AlicjaSuska** [sourcegraph/sourcegraph#12934]
Diffs on Sourcegraph are lacking + and - icons. We should add them to improve the experience for people with red-green color blindness.

Sourcegraph:
<img width="1131" alt="Screenshot 2020-08-11 at 16 36 02" src="https://user-images.githubusercontent.com/20326070/89912921-69653e00-dbf3-11ea-9034-50d1bf2dbe06.png">

GitHub:
<img width="1227" alt="Screenshot 2020-08-11 at 16 36 31" src="https://user-images.githubusercontent.com/20326070/89912936-6d915b80-dbf3-11ea-95ec-71c263e50cd6.png">

We can use the same solution as GitHub.

Concerns:
Should + and - be selectable or it would be annoying when copying? - on GitHub, they are not being selected. 

@eseliger - mentioning you as you were working on this component for Campaigns :)

**Comment by quinnkeast** [comment]
> Should + and - be selectable or it would be annoying when copying? - on GitHub, they are not being selected.

Would agree they should not be selected – they're visual affordances, not part of the content.

**Comment by eseliger** [comment]
It got lost when we introduced syntax highlighting, because in order to get it right, syntect needs the raw file content without any + or - signs, and we pick the final result together line-by-line.
I agree it makes sense, so we should find a solution to add them back, and I also agree they should _not_ be copied.

**Comment by AlicjaSuska** [comment]
Thank you for the feedback! @felixfbecker do you think we can take care of it any time soon?

**Comment by eseliger** [comment]
I can look into it this iteration
